pkg/controllers/dynakube/otelc/secret/reconciler.go

```diff
@@ -36,101 +36,101 @@ func NewReconciler(client client.Client, apiReader client.Reader, dk *dynakube.D
 
 func (r *Reconciler) Reconcile(ctx context.Context) error {
 	if r.dk.TelemetryIngest().IsEnabled() {
```

Comment:
the package is still called `secret`

and the methods in it are a mouthful, almost all are `....TelemetryIngestApiEndpointConfigMap`

so questions:
- is this a `telemetryIngest` specific logic?
  - if so, why not have it under `otelc/telemetryingest/endpoint` or `telemetryingest/configmap`
    - then you could simplify the method names => not everything would need to say `TelemetryIngest` (`ensureConfigMap` or `ensureEndpointConfigMap`)
- should the same be done with `otelc/service/`?  => `otelc/telemetryingest/service/`

Reply:
For now it is telemetryIngest specific, but it could get used more generic in the future. What about renaming it to `...OtlpApiEndpoint...`?